### ISSUE TITLE
Round Brackets don't work when using single check

### ISSUE THREAD

**Issue by KHOSH90** [mwarkentin/django-watchman#149]
WATCHMAN_CHECKS = (
    'xyxy.health.xyxy_check'
)
If I have a **single check** like the above, WATCHMAN_CHECKS variable is taken as a string and thus when frozenset is applied in get_checks method, the output is a **set of characters** and thus I get the below error in import_string method:-
{"error": 500, "message": "h doesn't look like a module path"}
This problem doesn't occur when there are multiple checks.
If I change the round brackets to square brackets, It works in **every case** since it takes the input of WATCHMAN_CHECKS as an array.
I think the documentation needs to be changed because this **warning** has to be added

**Comment by ulgens** [comment]
`("foo.bar")` is not a valid iterable for Python. It's same as "foo.bar". For tuples, correct syntax should be `("foo.bar", )`. I think this has nothing with django-watchman.

**Comment by KHOSH90** [comment]
Yes @ulgens you are correct. This has nothing to do with Django-watchman. But this deserves a place in the documentation.
